extensions/geode-modules-test/build.gradle

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+dependencies {
+  compileOnly ('org.apache.tomcat:catalina-ha:' + project.'tomcat6.version') {
+    exclude module: 'annotations-api'
+    exclude module: 'coyote'
+    exclude module: 'tribes'
+  }
+
+  compile(project(':extensions/geode-modules')) {
```

Comment:
Can we start preferring `implementation` over `compile` on these kinds of refactors in Gradle?

Reply:
Do we know how that will interact with the use of the older style compile/runtime method?